app/components/ui/PeoplePicker/PeoplePickerLogic.js

```diff
@@ -30,7 +30,7 @@ class PeoplePickerLogic extends React.Component {
     }
   }
 
-  isSelected(person) {
+  select(person) {
```

Comment:
This returns a boolean, and doesn't actually mutate anything. `isSelected` seems to be the correct name.

Reply:
Yeah I totally agree that `isXYZ` is the perfect name for a boolean value. But I think it's useful not to conflate the function which returns a boolean, with the return value itself.

[Line 93, below](https://github.com/elifesciences/elife-xpub/pull/1147/files/c4a5729b5d10a8fed94ef8f0367975485c11d1d9#diff-58e04d7ad244eea02c2d937aec254b5eR93) kind of shows what I mean:
`isSelected` is the return value of calling the `select` function with `person` as an input value

So there's a clear separation between the thing which generates the boolean (select) & the return value which _is_ a boolean (isSelected).

Does that make sense? Or am I missing something?

Reply:
- Pure functions, like `stringify` describe what the return value of the function would be
- Mutator functions like `update` have less emphasis on their names and more emphasis on the mutation being made 
- Accessor functions like `isConnected` are named after the state being queried for, and aren't expected to have any side effects

In this case, we're working with the third option. I see that the function is `PeoplePickerLogic.isSelected` which is adding to the confusion. `isPersonSelected(person)` would be clearer.